packages/client/src/providers/AuthProvider/index.js

```diff
@@ -56,14 +56,14 @@ const useAuthProvider = () => {
         }
 
         return Auth.createUserWithEmailAndPassword(email, password)
-            .then(() => {
-                const updateDisplayName = Auth.currentUser.updateProfile({displayName: userType});
-                return Promise.all([user.create(Auth.currentUser), updateDisplayName]);
+            .then(async () => {
+                await Auth.currentUser.updateProfile({displayName: userType});
```

Comment:
there is a small race condition where the displayName isn't set yet but the auth state has changed so rerenders happen, I don't think this fixes it though

I need to think harder about the correct ordering of things here